sdk/src/main/java/io/radar/sdk/Radar.kt

```diff
@@ -546,8 +540,17 @@ object Radar {
         val usage = "initialize"
         this.apiClient.getConfig(usage, false, object : RadarApiClient.RadarGetConfigApiCallback {
             override fun onComplete(status: RadarStatus, config: RadarConfig) {
-                locationManager.updateTrackingFromMeta(config?.meta)
-                RadarSettings.setFeatureSettings(context, config?.meta.featureSettings)
+                locationManager.updateTrackingFromMeta(config.meta)
```

Comment:
do we need to check the `RadarStatus` here?

Reply:
we should, it's fine if we don't though, I'll add in the check